test/trust_set_test.json

```diff
@@ -5,4 +5,4 @@
         }
     ],
     "transports": null
-}
+}
```

Comment:
CI unhappy here :(

Reply:
My IDE has different ideas on that file and keeps updating it. :-(